### DISCUSSION TITLE
Modifying the AraBERT-Trainer-Optuna-SA notebook

### DISCUSSION THREAD

**Discussion by ReemAlJunaid** [aub-mind/arabert#184] [Q&A]
How to modify [this notebook](https://colab.research.google.com/drive/1P9iQHtUH5KUbTVtp8B4-AopZzEEPE0lw?usp=sharing) instead of training the model on only one label, to be on multiple labels?

For example in my case, I want to classify the questions based on their expected answer type, and some of the questions are in fact composite questions, so they have multiple expected answer types.

Like: "هل استجاب الله تعالى لدعاء أيوب عليه السلام لرفع البلاء عنه؟ وبماذا أكرمه الله تعالى ولماذا ؟ "

**Comment by WissamAntoun** [comment]
check the discussion here to see if it helps https://github.com/aub-mind/arabert/discussions/81
